docker/workspace_entrypoint.sh

```diff
@@ -25,4 +25,8 @@ then
   export CYCLONEDDS_URI=file://${SCRIPT_DIR}/cyclonedds_profile.xml
 fi
 
+# TODO: there is probably a better way to do this
+# Install the angel system package
+pip install .
+
```

Comment:
We definitely don't want this: it will install anew any time a new container starts up. Package installation should be part of the Dockerfile. The docker-compose config mount should then mount the host package dir to the location installed into in the docker image.